### PR TITLE
[FrameworkBundle] Get KERNEL_CLASS through $_ENV too for KernelTestCase

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | 3.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/recipes/pull/170/files#diff-793b024840ee04ebdff5e04c2cd44f8cR12
| License       | MIT
| Doc PR        | -

I guess the check could be simplied in `master` with php7+

/cc @nicolas-grekas

### CODE REVIEW

**Review by nicolas-grekas — COMMENTED** [review]
KERNEL_DIR also should be updated

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php [hunk 1]
```diff
@@ -106,16 +106,17 @@ private static function getPhpUnitCliConfigArgument()
      */
     protected static function getKernelClass()
     {
-        if (isset($_SERVER['KERNEL_CLASS'])) {
-            if (!class_exists($class = $_SERVER['KERNEL_CLASS'])) {
+        if (isset($_SERVER['KERNEL_CLASS']) || isset($_ENV['KERNEL_CLASS'])) {
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php]: This should probably be fixed in 2.7.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php (hunk 1 above)
Comment (yceruto) [reply src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php]: Removed changes regarding to `KERNEL_DIR` var only, so `KERNEL_CLASS` was introduced in `3.3`.

Comment (yceruto) [reply src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php]: See https://github.com/symfony/symfony/pull/24115

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**